driver-sync/src/main/com/mongodb/client/internal/FindIterableImpl.java

```diff
@@ -161,6 +161,12 @@
         return this;
     }
 
+    @Override
+    public FindIterable<TResult> hintString(@Nullable final String hint) {
```

Comment:
I think there could be a test for this in `com.mongodb.client.internal.FindIterableSpecification#should build the expected findOperation`

Reply:
Done